### ISSUE TITLE
Detect super call without brackets

### ISSUE THREAD

**Issue by dpinol** [PyCQA/pylint#4008]
### Is your feature request related to a problem? Please describe
Sometimes people forget brackets after using super

### Describe the solution you'd like
I'd like a warning in the last line complaining of missing brackets after super
```python
# pylint: disable=no-self-use
class Base:
    def kk(self):
        print("Base.kk()")


class Derived(Base):
    def kk(self):
        super.kk()
        print("Derived.kk()")


a = Base()
a.kk()
b = Derived()
b.kk()

```

thanks
